packages/shared/src/components/MainLayout.tsx

```diff
@@ -235,6 +247,19 @@ export default function MainLayout({
           onRequestClose={() => setIsIncompleteRegistration(false)}
         />
       )}
+      {user && !user.infoConfirmed && (
+        <StyledModal
+          isOpen
+          onRequestClose={() => setIsIncompleteRegistration(false)}
```

Comment:
You can actually simply log them out here. It's most likely a ghost user, and they should not be able to continue browsing.

Reply:
Ah, true. We should just do that. Thanks for the note.